TERMSOFSERVICE.md

```diff
@@ -420,64 +457,68 @@ reasonable use, we have the right to block access to Tideways until such
-objection, we can determine whether the previous ToS in our contractual
-relationship with you shall remain valid or whether you shall receive the right
-to termination. Should you not object to the amended ToS within the objection
-period, they shall be considered to have been accepted.
+17.2. Tideways will inform you about changes in the ToS in textual form, so
+that you have at least two (2) weeks time to object to the changes. In case of
+an objection Tideways will determine if the previous ToS will stay valid in the
+contractual agreement between both parties or if you are granted a right to
+termination of the contract. In case you don’t object to the changed ToS within
+the period for objection they count as accepted.
 
 # 18. Confidentiality
 
-18.1. The contracting parties undertake to treat all information they gain
-knowledge of during the fulfillment of this contract as confidential and to use
-it only for contractually agreed purposes.
-
-18.2. The rights and obligations in accordance with this section concerning
+18.1. Protected or confidential information of the contracting parties for the
+purpose of this contract are all information concerning one party and have been
+transferred or disclosed to the other party and have been marked as
+confidential or have to be considered confidential after a consideration of all
+circumstances. Unattached to the prior provision all information is viewed as
+confidential that concerns business relations, information about business
+processes, know-how, basis for calculations, concepts, business plans, software
+algorithms, software concepts, product- and program specifications, strategies,
+sales- and marketing data as well as marketing concepts, personal data stated
+for our services (such as name, email address, payment details) and other trade
+secrets of the contracting parties.
+
+18.2. The confidentiality obligation of paragraph 18.1. may not apply for
+protected or confidential information, when
+
+1. (a) the information was disclosed prior through one party to the other party
+   or already been public;
+2. (b) after the discloser or transfer through one party the respective
+   information is made public without the participation of the other party or
+   unattached to the failure to keep the information confidential;
+3. (c) the information have been disclosed to the parties by a third party,
+   that has no direct or indirect confidentially obligation to the respective
+   party; or
+4. (d) the information have to be disclosed to the competent authorities.
+
+18.3. The rights and obligations in accordance with this section concerning
 confidentiality shall remain unaffected by a termination of this contract.
 
-# 19. Data Protection
+# 19. Data Protection, Encryption
 
-19.1. We processes your personal data only for a specific purpose and pursuant
-to the statutory provisions.
+19.1. We process your personal data only for a specific purpose and pursuant to
```

Comment:
Reference the Privacy policy and remove paragraphs that are also specified in the Privacy policy already.